### PR TITLE
Hook in old influxdb

### PR DESCRIPTION
**Pull request by Evanyl**
### Summary
Hook in old influxdb

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [ x] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [x ] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

File: software/tracksight/backend/app/influx_handler.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
+from dateutil.parser import parse
+from datetime import timedelta
+import requests
+
+URL = "https://us-east-1-1.aws.cloud2.influxdata.com"
+BUCKET = "testing"
+
+TEMP_TOKEN = "pyh_P66tpmkqnfB6IL73p1GVSyiSK_o5_fmt-1KhZ8eYu_WVoyUMddNsHDlozlstS8gZ0WVyuycQtQOCKIIWJQ=="
```
Comment (Evanyl) [inline software/tracksight/backend/app/influx_handler.py:8]: Going forward we'll need to do a few things:

1. Delete this token from grafana
2. Give developers a token that should be used for development, but not visible from the codebase
3. Create a new token and have it be an env variable in the deployed version

File: software/tracksight/backend/app/telemetry.py
Comment (myung03) [inline software/tracksight/backend/app/telemetry.py:1]: heads up, working on refactoring this code in my curr branch - spliting HTTP stuff into 1 file and Websocket stuff in another file :)

File: software/tracksight/frontend/src/app/components/query_data.tsx
Comment (myung03) [inline software/tracksight/frontend/src/app/components/query_data.tsx:1]: abstraction from query_data -> dropdown menu looks super good! chef's kiss

File: software/tracksight/frontend/src/app/components/graph.tsx
Comment (myung03) [inline software/tracksight/frontend/src/app/components/graph.tsx:1]: Right now when you have a long title or too many titles it compresses the graph quite a bit. Best solution for this I think could be moving legend to be horizontal; here's the link for the docs that I think could be helpful:
https://plotly.com/python/legend/

Screen Shot 2023-10-25 at 2 10 13 PM

File: software/tracksight/backend/app/influx_handler.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            "db": db,
+            "q": f"SHOW FIELD KEYS ON {db} FROM {measurement}",
+        }
+        response = requests.get(
+            f"{URL}/query",
+            headers=headers,
+            params=params
+        )
+
+        # lol
```
Comment (jpatch22) [inline software/tracksight/backend/app/influx_handler.py:70]: ncie

File: software/tracksight/backend/app/influx_handler.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        results = response.json()["results"][0]["series"][0]["values"]
```
[the remark this answers is not in the document]
Comment (Evanyl) [reply software/tracksight/backend/app/influx_handler.py:71]: Ah yea that happens when there's no data available at this time. I'll add a comment to implement proper error handling from frontend to backend later

File: software/tracksight/backend/app/telemetry.py
Comment (Evanyl) [reply software/tracksight/backend/app/telemetry.py:1]: let's gooo

File: software/tracksight/frontend/src/app/components/dropdown_menu.tsx [hunk 4]
```diff
@@ -1,58 +1,39 @@
 import { useState, useEffect } from 'react'; 
 import { DownOutlined, SmileOutlined } from '@ant-design/icons';
```
[the remark this answers is not in the document]
Comment (Evanyl) [reply software/tracksight/frontend/src/app/components/dropdown_menu.tsx:2]: good catch

File: software/tracksight/frontend/src/app/components/graph.tsx
Comment (Evanyl) [reply software/tracksight/frontend/src/app/components/graph.tsx:1]: Incredible catch!

File: software/tracksight/frontend/src/app/components/query_data.tsx [hunk 5]
```diff
@@ -4,21 +4,65 @@ import { useEffect, useState } from 'react';
...
 
 const QueryData = (props) => {
+    const [measurement, setMeasurement] = useState([]);
+    const [allMeasurements, setAllMeasurements] = useState([]);
+
+    const [fields, setFields] = useState([]);
+    const [allFields, setAllFields] = useState([]);
+
     const [signals, setSignals] = useState([]);
     const [avail, setAvail] = useState([]);
```
[the remark this answers is not in the document]
Comment (Evanyl) [reply software/tracksight/frontend/src/app/components/query_data.tsx:17]: I think it's useful for maybe debugging purposes? I can remove it for now though, and if we want it back later it won't take long

**Review by myung03 — APPROVED** [review]

**Pull request merged**